docker/env/version

```diff
@@ -1 +1 @@
-0.59
+0.60
```

Comment:
when working on PR and setting docker version please set it to  not real version like `0.60-add-tc` since if you are will change the Dockerfile after resolving comments there will be two different versions of `0.60` image (when we run CI we build the new version of the image)

Reply:
got it